src/api/QdmCalculationService.ts

```diff
@@ -157,10 +157,15 @@ export class QdmCalculationService {
     }
   };
 
+  // episodeResults.observation_values holds results for all observations associated to that episode from all groups.
+  // It is assumed that the observation values are stored sequentially, so based on groupIndex, we are getting observations results for that particular group
+  // Example: Episode-xyz has observation_values [3,56,8], For this particular episode, group-1's actual value is 3, group-2's actual value is 56 and so on...
+  // Observation_Values are ignored if MSRPOPL == 1 & MSRPOPLEX == 0, since the execution results might contain the value, even though it is not a valid scenario.
```

Comment:
you mean `MSRPOPL == 1 & MSRPOPLEX == 1`?

Reply:
oops yeah, that not ideal, I will rephrase it